src/components/Head.jsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable react/jsx-indent-props */
+import React from 'react';
+import './styles/Crypto.css';
+import 'bootstrap/dist/js/bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Carousel from 'react-bootstrap/Carousel';
+import bitcoin from './assets/bitcoin.png';
+import cardano from './assets/cardano.png';
+import ethereum from './assets/ethereum.png';
+import tether from './assets/tether.png';
+
+function Head() {
+  return (
+    <div className="div-class">
```

Comment:
- [ ] I think it will be better if you wrap this section in a `<header>` tag since it is the introductory content of your app. Doing this will improve your website accessibility 👍🏽